contexts/v1.jsonld

```diff
@@ -2,51 +2,54 @@
   "@context": {
     "@protected": true,
 
-    "StatusList2021Credential": {
-      "@id":
-        "https://w3id.org/vc/status-list#StatusList2021Credential",
-      "@context": {
-        "@protected": true,
-
-        "id": "@id",
-        "type": "@type",
+    "BitstringStatusListCredential": "https://www.w3.org/ns/credentials/status-list#BitstringStatusListCredential",
 
-        "description": "http://schema.org/description",
-        "name": "http://schema.org/name"
-      }
-    },
-
-    "StatusList2021": {
-      "@id":
-        "https://w3id.org/vc/status-list#StatusList2021",
+    "BitstringStatusList": {
+      "@id": "https://www.w3.org/ns/credentials/status-list#BitstringStatusList",
       "@context": {
         "@protected": true,
 
         "id": "@id",
         "type": "@type",
 
         "statusPurpose":
-          "https://w3id.org/vc/status-list#statusPurpose",
-        "encodedList": "https://w3id.org/vc/status-list#encodedList"
+          "https://www.w3.org/ns/credentials/status-list#statusPurpose",
+        "encodedList":
+          "https://www.w3.org/ns/credentials/status-list#encodedList",
+        "ttl": "https://www.w3.org/ns/credentials/status-list#ttl",
+        "reference": "https://www.w3.org/ns/credentials/status-list#reference",
+        "size": "https://www.w3.org/ns/credentials/status-list#size",
```

Comment:
This is actually the size of the status. This is misnamed/mismodeled and should probably be `statusSize`.

Reply:
This is being tracked in issue #113 (but we might as well make the change here).

Reply:
Fixed in 35860acfc10c8ed7db692e1ef52d43655d770176.